datafusion/proto/src/logical_plan/to_proto.rs

```diff
@@ -1052,11 +1054,6 @@ impl TryFrom<&Expr> for protobuf::LogicalExprNode {
                     })),
                 }
             }
-
```

Comment:
As a bonus, qualified wildcards can now be serialized to/from proto -- I doubt this feature is widely used, but it is nice to avoid the rough edge I think